tests/common.py

```diff
@@ -477,6 +478,18 @@ def mock_device_registry(hass, mock_entries=None, mock_deleted_entries=None):
     return registry
 
 
+def mock_label_registry(
```

Comment:
Instead of adding this mock, or perhaps in addition to adding this mock if it's actually useful, update `async_test_home_assistant` to load the label registry unless the `load_registries` fixture is `False`, tests can then just call `label_registry.async_get`.

Reply:
Adjusted it in my local version. Marking this as resolved for now, to clean up my view on this PR.